sources/google_sheets/README.md

```diff
@@ -1,133 +1,62 @@
-# Google sheets
+# Google Sheets
 
-Here you will find a setup guide for the [Google sheets](https://www.google.com/sheets/about/) source with an example pipeline.
+This verified source can be used to load data from a [Google Sheets](https://www.google.com/sheets/about/) workspace onto a destination of your choice.
 
-## Set up account
-1.  Set up a Google account
-2.  Login to google sheets using your google credentials.
+| Endpoints | Description |
+| --- | --- |
+| Tables | tables of the spreadsheet, tables have same name as individual sheets |
+| Named ranges | loaded as a separate column with an automatically generated header. |
+| Merged cells | retains only the cell value that was taken during the merge (e.g., top-leftmost), and every other cell in the merge is given a null value. |
```

Comment:
all rows in this tables should be ended with dot "." or all of them should be without it

Reply:
please check now.